### ISSUE TITLE
Version 1.8 cannot close the gaze point in the studio, just return to 1.7

### ISSUE THREAD

**Issue by ikvmao** [IllusionMods/IllusionFixes#9]
1.8MakerOptimizations does not have (Disable camera target (white focus ring)) option, 1.7 does. .

**Comment by ManlyMarco** [comment]
This functionality has been moved to a new plugin cameratargetfix in https://github.com/Keelhauled/KeelPlugins#cameratargetfix

You can still use the in-game option to disable the target in maker, press F2 to see the hotkey for it.

**Comment by ikvmao** [comment]
The cameratargetfix plugin seems to be invalid for studio gaze points. I can only use 1.7.

**Comment by Keelhauled** [comment]
CameraTargetFix uses the button in the bottom left that disables the camera target. Settings in the config are useless now.

**Comment by ikvmao** [comment]
I was using CameraTarget Fix and IllusionFixes version 1.8. Clicking the gaze point button in the lower left corner of the studio was useless, so I just retreated to 1.7.

**Comment by Keelhauled** [comment]
You're using the CameraTargetFix from the KeelPlugins repo? Remove that, they seem to break each other.

**Comment by ikvmao** [comment]
Can't two exist at the same time? I told you yesterday that you did n’t say that feature was migrated to CameraTargetFix? Well, wait for me to try it.
